django_x509/base/admin.py

```diff
@@ -2,11 +2,13 @@
 from django.conf.urls import url
 from django.contrib.admin import ModelAdmin
 from django.contrib.admin.templatetags.admin_static import static
+from django.http import HttpResponse
+from django.shortcuts import get_object_or_404
 from django.urls import reverse
 from django.utils.html import format_html
 from django.utils.translation import ugettext_lazy as _
 
-from django_x509.views import crl
```

Comment:
@atb00ker have you tried using `django_x509.base.views import crl` instead of `django_x509.views import crl` i think that can work.